src/middlewared/middlewared/alembic/initial.sql

```diff
@@ -189,8 +189,8 @@ CREATE TABLE IF NOT EXISTS "services_nfs" ("id" integer NOT NULL PRIMARY KEY AUT
 INSERT INTO services_nfs VALUES(1,4,0,0,0,'False',0,NULL,NULL,NULL,'False','True','False','');
 CREATE TABLE IF NOT EXISTS "services_dynamicdns" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "ddns_provider" varchar(120) NOT NULL, "ddns_domain" varchar(120) NOT NULL, "ddns_username" varchar(120) NOT NULL, "ddns_password" varchar(120) NOT NULL, "ddns_checkip_ssl" bool NOT NULL, "ddns_checkip_server" varchar(150) NOT NULL, "ddns_checkip_path" varchar(150) NOT NULL, "ddns_ssl" bool NOT NULL, "ddns_custom_ddns_server" varchar(150) NOT NULL, "ddns_custom_ddns_path" varchar(150) NOT NULL, "ddns_period" integer NOT NULL);
 INSERT INTO services_dynamicdns VALUES(1,'dyndns','','admin','DU8t+PUH4e2koeM3Pnquy+kNAEjdpCy0E0S81cb0GN5Naunfmr9B/w==',0,'','',0,'','',300);
-CREATE TABLE IF NOT EXISTS "services_ftp" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "ftp_port" integer unsigned NOT NULL, "ftp_clients" integer unsigned NOT NULL, "ftp_ipconnections" integer unsigned NOT NULL, "ftp_loginattempt" integer unsigned NOT NULL, "ftp_timeout" integer unsigned NOT NULL, "ftp_rootlogin" bool NOT NULL, "ftp_onlyanonymous" bool NOT NULL, "ftp_anonpath" varchar(255) NULL, "ftp_onlylocal" bool NOT NULL, "ftp_banner" text NOT NULL, "ftp_filemask" varchar(3) NOT NULL, "ftp_dirmask" varchar(3) NOT NULL, "ftp_fxp" bool NOT NULL, "ftp_resume" bool NOT NULL, "ftp_defaultroot" bool NOT NULL, "ftp_ident" bool NOT NULL, "ftp_reversedns" bool NOT NULL, "ftp_masqaddress" varchar(120) NOT NULL, "ftp_passiveportsmin" integer unsigned NOT NULL, "ftp_passiveportsmax" integer unsigned NOT NULL, "ftp_localuserbw" integer unsigned NOT NULL, "ftp_localuserdlbw" integer unsigned NOT NULL, "ftp_anonuserbw" integer unsigned NOT NULL, "ftp_anonuserdlbw" integer unsigned NOT NULL, "ftp_tls" bool NOT NULL, "ftp_tls_policy" varchar(120) NOT NULL, "ftp_tls_opt_allow_client_renegotiations" bool NOT NULL, "ftp_tls_opt_allow_dot_login" bool NOT NULL, "ftp_tls_opt_allow_per_user" bool NOT NULL, "ftp_tls_opt_common_name_required" bool NOT NULL, "ftp_tls_opt_enable_diags" bool NOT NULL, "ftp_tls_opt_export_cert_data" bool NOT NULL, "ftp_tls_opt_no_cert_request" bool NOT NULL, "ftp_tls_opt_no_empty_fragments" bool NOT NULL, "ftp_tls_opt_no_session_reuse_required" bool NOT NULL, "ftp_tls_opt_stdenvvars" bool NOT NULL, "ftp_tls_opt_dns_name_required" bool NOT NULL, "ftp_tls_opt_ip_address_required" bool NOT NULL, "ftp_ssltls_certificate_id" integer NULL REFERENCES "system_certificate" ("id"), "ftp_options" text NOT NULL);
```

Comment:
You can't do that, you need to generate a migration

Reply:
removed this change and converted to alembic migration